src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
Please, don't change the method signature
```suggestion
    public User findByEmail(String email) {
```

Reply:
Fixed.